index.html

```diff
@@ -5,7 +5,7 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link rel="stylesheet" href="./style.css">
-        <title>Mobile Version</title>
+        <title>Ogaga's Portfolio</title>
```

Comment:
- [ ] Please, since you found an issue here and you made a change, now you need to link the corresponding line(line 8) of the code in the Pull Request changes.

Reply:
Hi @OLIPLICHE. Thank you for pointing this out but I have already added the changes and stated them in the pull request description.
If there are any other changes or if I haven't made the required changes please let me know and please explain further

Reply:
Okay